docsets/Agora_Java/docset.json

```diff
@@ -9,7 +9,14 @@
     "aliases": ["agora java"],
 
     "specific_versions": [
-        
+
+        {
+            "_comment": "This is Agora Java API Reference for Android v2.9.0"
+        },
+        {
+            "version": "2.9.0",
+            "archive": "versions/2.9.0/Agora_Java.tgz",
```

Comment:
You need to actually add this archive. Moving/renaming the txt file does nothing.

Reply:
Fixed.